### PR TITLE
Add note about snapshot best practice

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Adds information about differences between backups on Pro Production, Staging, and Integration environments and clarifies that restoring from automatic backups requires a support ticket for Production and Staging environments.

## Affected DevDocs pages

- https://devdocs.magento.com/cloud/architecture/pro-architecture.html

### CODE REVIEW

File: src/cloud/architecture/pro-architecture.md [hunk 1]
```diff
@@ -178,6 +178,9 @@ Weeks 12 to 22 | One backup per month
 -  **Recovery Point Objective (RPO)**—is 1 hour for the first 24 hours; after which, the RPO is 6 hours (maximum time to last backup).
 -  **Recovery Time Objective (RTO)**—depends on the size of the storage. Large EBS volumes take more time to restore.
 
+{:.tip}
+On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{  site.baseurl }}/cloud/project/project-webint-snap.html#db-dumpump] and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot of your environment as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management](https://devdocs.magento.com/cloud/project/project-webint-snap.html).
```
Comment (meker12) [inline src/cloud/architecture/pro-architecture.md:182]:
```suggestion
On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{  site.baseurl }}/cloud/project/project-webint-snap.html#db-dumpump] and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management](https://devdocs.magento.com/cloud/project/project-webint-snap.html).
```

File: src/cloud/architecture/pro-architecture.md [hunk 2]
```diff
@@ -178,6 +178,9 @@ Weeks 12 to 22 | One backup per month
 -  **Recovery Point Objective (RPO)**—is 1 hour for the first 24 hours; after which, the RPO is 6 hours (maximum time to last backup).
 -  **Recovery Time Objective (RTO)**—depends on the size of the storage. Large EBS volumes take more time to restore.
 
+{:.tip}
```
Comment (hguthrie) [inline src/cloud/architecture/pro-architecture.md:181]: This does nothing.

Maybe need to use: 
```suggestion
{:.bs-callout-tip}
```

File: src/cloud/architecture/pro-architecture.md [hunk 3]
```diff
@@ -178,6 +178,9 @@ Weeks 12 to 22 | One backup per month
...
+On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{  site.baseurl }}/cloud/project/project-webint-snap.html#db-dumpump] and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management](https://devdocs.magento.com/cloud/project/project-webint-snap.html).
```
Comment (hguthrie) [inline src/cloud/architecture/pro-architecture.md:182]: Broken links

```suggestion
On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{  site.baseurl }}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management](https://devdocs.magento.com/cloud/project/project-webint-snap.html).
```

**Review by hguthrie — CHANGES_REQUESTED** [review]
Couple of broken things to fix.

File: src/cloud/architecture/pro-architecture.md [hunk 4]
```diff
@@ -178,6 +178,9 @@ Weeks 12 to 22 | One backup per month
 -  **Recovery Point Objective (RPO)**—is 1 hour for the first 24 hours; after which, the RPO is 6 hours (maximum time to last backup).
 -  **Recovery Time Objective (RTO)**—depends on the size of the storage. Large EBS volumes take more time to restore.
 
+{:.bs-callout-tip}
+On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{  site.baseurl }}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management](https://devdocs.magento.com/cloud/project/project-webint-snap.html).
```
Comment (meker12) [inline src/cloud/architecture/pro-architecture.md:182]:
```suggestion
On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{site.baseurl}}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management]({{ site.baseurl}}/cloud/project/project-webint-snap.html).
```

File: src/cloud/architecture/pro-architecture.md [hunk 5]
```diff
@@ -178,6 +178,9 @@ Weeks 12 to 22 | One backup per month
 -  **Recovery Point Objective (RPO)**—is 1 hour for the first 24 hours; after which, the RPO is 6 hours (maximum time to last backup).
 -  **Recovery Time Objective (RTO)**—depends on the size of the storage. Large EBS volumes take more time to restore.
 
+{:.bs-callout-tip}
+On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{site.baseurl}}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management]({{ site.baseurl}}/cloud/project/project-webint-snap.html).
```
Comment (meker12) [inline src/cloud/architecture/pro-architecture.md:182]:
```suggestion
On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{site.baseurl}}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management]({{site.baseurl}}/cloud/project/project-webint-snap.html).
```

**Review by hguthrie — APPROVED** [review]

**Comment by meker12** [comment]
running tests

File: src/cloud/architecture/pro-architecture.md [hunk 6]
```diff
@@ -178,6 +178,9 @@ Weeks 12 to 22 | One backup per month
 -  **Recovery Point Objective (RPO)**—is 1 hour for the first 24 hours; after which, the RPO is 6 hours (maximum time to last backup).
 -  **Recovery Time Objective (RTO)**—depends on the size of the storage. Large EBS volumes take more time to restore.
 
+{:.bs-callout-tip}
+On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{site.baseurl}}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.htm#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management]({{site.baseurl}}/cloud/project/project-webint-snap.html).
```
Comment (meker12) [inline src/cloud/architecture/pro-architecture.md:182]:
```suggestion
On Pro Staging and Production environments, you must submit a support ticket to restore an environment from an automatic backup. You can backup the database and code for your Production and Staging environments using CLI commands. See [Dump your database]({{site.baseurl}}/cloud/project/project-webint-snap.html#db-dump) and [bin/magento setup:backup]({{site.baseurl }}/guides/v2.4/reference/cli/magento-commerce.html#setupbackup). For Integration environments, we highly recommend that you create a snapshot as a first step after accessing your {{site.data.var.ece}} project and before applying any major changes. See [Snapshots and backup management]({{site.baseurl}}/cloud/project/project-webint-snap.html).
```

**Comment by meker12** [comment]
running tests

**Pull request merged**